app/assets/stylesheets/components/_card_list.css.sass

```diff
@@ -26,6 +26,12 @@
       &.ng-enter-active
         animation-name: bounceInLeft
 
+.CardList-card.is-closed
```

Comment:
Yes the Discussion is the "interior" of the discussion, where you can see the description and the comments. This CardList is just the list of all the discussion, so the .is-closed is well placed where.
You can merge @frmendes